### ISSUE TITLE
Does this work with Jazzy??

### ISSUE THREAD

**Issue by dp1140a** [tiiuae/rclgo#23]
Does this work with Jazzy??

If not what needs to be changed to get it working?

**Comment by dp1140a** [comment]
Anyone???

**Comment by feygina** [comment]
I think it doesn't :(

**Comment by jan-ati** [comment]
See my response for getting this working with Iron for ideas on required changes: https://github.com/tiiuae/rclgo/issues/24

**Comment by dp1140a** [comment]
There is a fork of this that works on Jazzy.  [https://github.com/PolibaX/rclgo](https://github.com/PolibaX/rclgo) Could do a diff.  I actually had to backport this to Humble as I cant run jazzy yes on my laptop since jazzy requires 24.04 and I'm on PopOS 22.04.  But I got that working as well.  Maybe we can do a PR to sync them all?

**Comment by jan-ati** [comment]
Makes sense to me. I'm planning on doing a PR (at least into a `ros2-iron` branch on this repo)  once the Pipeline issues are fully resolved on my fork.

**Comment by richiejp** [comment]
There is also https://github.com/ATIinc/rclgo which has received some attention recently. I sent a PR for an unrelated issue.

**Comment by dp1140a** [comment]
I forked it myself and have been doing a lot of work on it:
https://github.com/merlindrones/rclgo

Im trying to get parity with rclpy and rclcpp

So far I have added support for params and qos
